pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -344,13 +344,21 @@ def on_save_checkpoint(
             "best_model_path": self.best_model_path,
             "current_score": self.current_score,
```

Comment:
current_score does not get loaded? I wonder if there was a reason for that

Reply:
current_score changes at every iteration and is not used for anything else. Does it need a reload?

Reply:
Yeah makes sense, I guess it just gets saved to the checkpoint so that it can be inspected if needed.  👍